Functions/Domain/RuleHistoryCache.cs

```diff
@@ -9,5 +9,8 @@ public class RuleHistoryCache : BaseEntity
         public string ChangedAtDateTime { get; set; }
         public string ChangedByDisplayName { get; set; }
         public string ChangedByEmail { get; set; }
+        public string CreatedAtDateTime { get; set; }
```

Comment:
Might be nice to store these values as Datetime if we ever want to do a query that filters/sorts on it.